Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM golang:1.16.2 AS builder
+ENV GOPROXY=direct GOSUMDB=off
+WORKDIR /go/src/project
+ADD *.go go.* ./
+ADD cmd cmd
+ADD crypto crypto
+ADD fsutil fsutil
+ADD logger logger
+ADD service service
+ADD version version
+ADD wallet wallet
```

Comment:
Why not add all the folders at once? doing that means updating the image when we add new packages.